### ISSUE TITLE
Identifiers: help understanding the (HTML) report

### ISSUE THREAD

**Issue by marcelstoer** [dependency-check/DependencyCheck#4309]
I have just been through "How to read the report" and "Suppressing false positives" (again) but I still don't understand what I am seeing in the HTML report excerpt below.

There are a number of CVEs published for Apache James - which my project has no dependency to. However, somehow the identifier for Apache James is linked to `commons-validator` and `commons-validator` and thus showing up in my report.

It is great the report contains a suppress button, thanks for that! I am not a 100% sure though what exactly I'd be suppressing with this:

```xml
<suppress>
   <notes><![CDATA[
   file name: commons-digester-2.1.jar
   ]]></notes>
   <packageUrl regex="true">^pkg:maven/commons\-digester/commons\-digester@.*$</packageUrl>
   <cpe>cpe:/a:apache:james</cpe>
</suppress>
```

<img width="1019" alt="Screen Shot 2022-04-05 at 12 34 13" src="https://user-images.githubusercontent.com/624195/161736300-16413c7f-c93b-466c-a527-6a696d0ecba0.png">

**Comment by mprins** [comment]
This should be fixed in the current release, see #4148

**Comment by marcelstoer** [comment]
Thank you so much! Looks like my CI configuration that should ensure we always use the latest dependency-check version is buggy. Just noticed my report was created with 7.0.0.
